src/components/Users/UserService.tsx

```diff
@@ -31,7 +32,43 @@ export function sanitizeUser(user) {
 
 
 
+export function login(loginDetails: { email: string, password: string }): Promise<User> {
+    return new Promise((resolve, reject) => {
+        HttpService.http({
+            method: 'POST',
+            url: '/api/login',
+            body: loginDetails
+        }).then((response) => {
+            if (response.status === 200) {
+                let user = response.data.user;
+
+                StorageService.save('user', user);
+                StorageService.save('auth-token-manually-verified', true, {session: true});
```

Comment:
This clears each time the user closes the window. I envisioned having a manual auth token verification endpoint that would require that the user's auth token was manually verified every time they come back for a session, and then it would randomly verify the auth token every x times that they made a request to a secure endpoint through the middleware.